[2024-10-01T08:53:00.856+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-01T08:53:00.958+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_id_1.branch_condition manual__2024-10-01T08:52:43.360341+00:00 [queued]>
[2024-10-01T08:53:01.042+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_id_1.branch_condition manual__2024-10-01T08:52:43.360341+00:00 [queued]>
[2024-10-01T08:53:01.054+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-01T08:53:01.131+0000] {taskinstance.py:2888} INFO - Executing <Task(BranchPythonOperator): branch_condition> on 2024-10-01 08:52:43.360341+00:00
[2024-10-01T08:53:01.156+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1416) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-01T08:53:01.163+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag_id_1', 'branch_condition', 'manual__2024-10-01T08:52:43.360341+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/dag_id_1.py', '--cfg-path', '/tmp/tmp0grrbdmo']
[2024-10-01T08:53:01.171+0000] {standard_task_runner.py:105} INFO - Job 146: Subtask branch_condition
[2024-10-01T08:53:01.166+0000] {standard_task_runner.py:72} INFO - Started process 1455 to run task
[2024-10-01T08:53:02.192+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_id_1.branch_condition manual__2024-10-01T08:52:43.360341+00:00 [running]> on host 7a112cbb4731
[2024-10-01T08:53:03.838+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_id_1' AIRFLOW_CTX_TASK_ID='branch_condition' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T08:52:43.360341+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-01T08:52:43.360341+00:00'
[2024-10-01T08:53:03.868+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-01T08:53:03.888+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-01T08:53:03.892+0000] {logging_mixin.py:190} INFO - Current task name:branch_condition state:running start_date:2024-10-01 08:53:00.959850+00:00
[2024-10-01T08:53:03.900+0000] {logging_mixin.py:190} INFO - Dag name:dag_id_1 and current dag run status:running
[2024-10-01T08:53:03.905+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-01T08:53:03.914+0000] {baseoperator.py:405} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-10-01T08:53:03.919+0000] {python.py:240} INFO - Done. Returned value was: insert_new_row
[2024-10-01T08:53:03.934+0000] {branch.py:38} INFO - Branch into insert_new_row
[2024-10-01T08:53:03.991+0000] {skipmixin.py:230} INFO - Following branch ('insert_new_row',)
[2024-10-01T08:53:04.129+0000] {skipmixin.py:278} INFO - Skipping tasks [('create_table', -1)]
[2024-10-01T08:53:04.575+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-01T08:53:04.594+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_id_1, task_id=branch_condition, run_id=manual__2024-10-01T08:52:43.360341+00:00, execution_date=20241001T085243, start_date=20241001T085300, end_date=20241001T085304
[2024-10-01T08:53:04.705+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-01T08:53:04.709+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-01T08:53:04.713+0000] {logging_mixin.py:190} INFO - Dag name:dag_id_1 queued_at:2024-10-01 08:52:43.445613+00:00
[2024-10-01T08:53:04.721+0000] {logging_mixin.py:190} INFO - Task hostname:7a112cbb4731 operator:BranchPythonOperator
[2024-10-01T08:53:04.780+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-01T08:53:04.889+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
